### ISSUE TITLE
Update Readme

### ISSUE THREAD

**Issue by robinzigmond** [vivithemage/do-wp#30]
I noticed that the Readme is somewhat out of date, at least as regards telling users how to use the application. At the moment this is found only in the "quick start" section - but this only refers to the "publish" option, with no reference to the alternative "initialize" process.

So this needs to be rewritten to clearly state that there are 2 different things you can do, and how each works.

In addition to this, the following changes will also be needed:
- it currently states that, after publication, you can paste the IP address into the browser to see the site; this no longer works because the tool updates the wordpress site_url to the URL provided, which means that it will redirect to the desired URL, and this does nothing until the DNS is updated. (I know that there is another issue currently open to automate this.)
- the change I am currently working on provides defaults for the database fields in the "initialize" process, providing relevant environment variables are set on the user's machine - this should be mentioned in the instructions for this process, as these can be set once to greatly simplify the procedure from the user's point of view (providing of course the host, username and db are not often changed)

While we're at it the entire README probably wants updating. And since we're making this open-source software, we don't want to make everything too specific to Thrive, even though we're obviously going to be the main (only?) users at the moment. So I can hopefully automate the DNS to work through cloudflare, but I'm not sure how to make this then usable for potential other users who use a different DNS provider.

**Comment by vivithemage** [comment]
Yes, the readme could definitely do with updating so edit as you see fit and we can review.

For the first bullet point, I'd suggest we abandon the IP part and integrate with cloudflare. I know it's not generic but if anyone comes along and says, "we don't use cloudflare, can we use an IP" we could modify to cater for that. But I'd be amazed if that happened (I'm not aware of anyone else using this but us).

For the second point, pre filled options is definitely the way forward. However, it's probably best to pull this from a config file in the root of the project

Yes, we should try to make it as generic as possible. On that note a checkbox that says 'use cloudflare' and then a location to add an api key (perhaps a button next to 'about' called options) may be the best option.

Of course all open to discussion. Just let me know what you think.

Thanks,
Reza
